BE/deploy/scripts/blue-green-deploy.sh

```
@@ -0,0 +1,152 @@
+    DEPLOY_ENV="blue"
+    CURRENT_PORT=8080
+else
+    if [[ "$CURRENT_API_CONTAINER" == *"blue"* ]]; then
+        echo -e "${COLOR_BLUE}Current: BLUE, Deploying: GREEN${COLOR_RESET}"
+        DEPLOY_ENV="green"
+        CURRENT_PORT=8080
+        NEW_PORT=8081
+    else
+        echo -e "${COLOR_GREEN}Current: GREEN, Deploying: BLUE${COLOR_RESET}"
+        DEPLOY_ENV="blue"
+        CURRENT_PORT=8081
```

Shellcheck (0.11.0)

[warning] CURRENT_PORT appears unused. Verify use (or export if used externally).

(SC2034)